packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -393,6 +400,34 @@ export const HelpCenterContactForm = () => {
 		}
 	};
 
+	const getSupportedLanguages = ( supportType: string, locale: string ) => {
+		const isLiveChatLanguageSupported = (
+			config( 'livechat_support_locales' ) as Array< string >
+		 ).includes( locale );
+
+		const isLanguageSupported = ( config( 'upwork_support_locales' ) as Array< string > ).includes(
+			locale
+		);
+
+		switch ( supportType ) {
+			case SUPPORT_HAPPYCHAT:
+				return ! isLiveChatLanguageSupported;
+
+			case SUPPORT_TICKET:
+			case SUPPORT_CHAT_OVERFLOW:
+			case SUPPORT_UPWORK_TICKET:
+				return ! isLanguageSupported && ! [ 'en', 'en-gb' ].includes( locale );
+
+			default:
+				return false;
+		}
+	};
+
+	const shouldShowHelpLanguagePrompt = () => {
+		const supportType = getSupportVariationFromMode( mode );
```

Comment:
I think this is an extra step. You can probably just use the `mode` rather than trying to gather the `supportVariation`. We only have these options we need to account for `type Mode = 'CHAT' | 'EMAIL' | 'DIRECTLY' | 'FORUM';` and directly is on the way out the door.